lib/rules/web/admin_console/4.5/catalog.xyaml

```diff
@@ -90,6 +90,10 @@ filter_by_template_type:
   params:
     type: template
   action: filter_by_type
+filter_by_operatorbacked_type:
+  params:
+    type: cluster-service-version
+  action: filter_by_type
```

Comment:
For ocp-26887 on catalog page, should remove filter to show all catalog items.

Reply:
If this is used to not selecting, then I would use `show_all_catalog_item` as name

Reply:
No, it's just control the filter for operatorbacked type.